### PR TITLE
docker: handle non-user generated IOErrors in zap-baseline

### PR DESCRIPTION
**Pull request by g-k**
refs: https://github.com/zaproxy/zaproxy/issues/3462 similar to: https://github.com/zaproxy/zaproxy/pull/3347

This won't fix the root cause of #3462 but it should let us surface the underlying error.

### CODE REVIEW

File: build/docker/zap-baseline.py [hunk 1]
```diff
@@ -577,9 +577,13 @@ def main(argv):
...
-    print("ERROR " + str(strerror))
-    logging.warning ('I/O error(' + str(errno) + '): ' + str(strerror))
+  except IOError as e:
+    if hasattr(e, 'args') and len(e.args) > 1:
+      errno, strerror = e
+      print("ERROR " + str(strerror))
+      logging.warning ('I/O error(' + str(errno) + '): ' + str(strerror))
+    else:
+      print("ERROR %s" % e)
     dump_log_file(cid)
```
Comment (psiinon) [inline build/docker/zap-baseline.py]: Missing a logging.warning(..) call in the else path?

**Comment by psiinon** [comment]
The 'space' changes make it difficult to see whats actually changed vs formatting tweaks. Formatting changes are good but they should be made in separate PRs as per https://github.com/zaproxy/zaproxy/wiki/DevGuidelines :)

**Comment by g-k** [comment]
Ugh, yeah that's just my editor config. I'll make a separate PR for that change.

**Comment by thc202** [comment]
Thanks @g-k. Pull request description could be updated to fix #3462, with these changes the output now shows that ZAP was not able to start because of wrong command line arguments.

**Pull request merged**